.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,7 +5,7 @@ Checklist:
 - [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
 - [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
 - [ ] My pull request targets the `main` branch of freeCodeCamp.
-- [ ] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.
```

Comment:
Are we removing this in favour of the bot reminders?

Reply:
Yes.